packages/react-components/src/dot/tests/chromatic/Dot.chroma.jsx

```diff
@@ -24,7 +24,7 @@ stories()
     .add("disabled", () =>
         <Stack>
             <Dot disabled></Dot>
-            <Dot disabled>Habitable</Dot>
+            <Dot color="hsl(25, 69%, 41%)" disabled>Habitable</Dot>
```

Comment:
Dot chromatic "color" tests should be updated to have a color value with every supported color format.

- CSS class
- hexa
- hsl
- etc..

Reply:
> Since you changed the spacing between the trigger and the popover + tooltip, should you also do it for Menu? Maybe Select & Autocomplete too?

this was due to the arrow being too close to the trigger, menu/select and autocomplete do not have those arrows.